### PR TITLE
feat: add license classifier to PyPI package

### PR DESCRIPTION
**Pull request by RobPasMue**
Having the license classifier in PyPI is also important so that it is uploaded as part of the package's metadata. Otherwise, when querying it to PyPI this information is not shown in the package's metadata.

- [X] I understand that this repository is auto-generated and my pull request may not be merged

### CODE REVIEW

**Comment by rattrayalex** [comment]
Thanks @RobPasMue , this looks reasonable to me. It may take a bit to merge since we'll need to update our codegen to reflect.

**Review by schnerd — APPROVED** [review]

**Comment by schnerd** [comment]
(Approving but will defer to you on when to merge Alex)

**Comment by RobertCraigie** [comment]
Thanks!

**Pull request merged**
